airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/destination/buffered_stream_consumer/BufferedStreamConsumer.java

```diff
@@ -116,12 +114,38 @@ public BufferedStreamConsumer(final Consumer<AirbyteMessage> outputRecordCollect
     this.stateManager = new DefaultDestStateLifecycleManager();
```

Comment:
this constructor should call the visiblefortesting constructor, i.e. `this(outputRecordCollector, onStart, ....., Duration.ofMinutes(15));` rather than repeating all the field assignments

and then you can declare `private final Duration bufferFlushFrequency;` without an explicit initial value